src/test/java/org/broadinstitute/hellbender/engine/spark/datasources/ReadsSparkSinkUnitTest.java

```diff
@@ -27,9 +31,20 @@
 import java.util.List;
 
 public class ReadsSparkSinkUnitTest extends BaseTest {
+    private MiniDFSCluster cluster;
```

Comment:
Class-level variables in test classes are problematic in theory if we ever want to switch to method-level test suite parallelism -- is it too costly to spin up a separate cluster for each test case?